_config.yml

```diff
@@ -517,6 +517,10 @@ math:
 # To install 3.x: https://github.com/theme-next/theme-next-fancybox3
 fancybox: false
 
+# A JavaScript library for zooming images like Medium.
+# Dependencies: https://github.com/theme-next/theme-next-mediumzoom
+mediumzoom: false
```

Comment:
I think we should let user know that this plugin should not be enabled when fancybox is enabled.

Reply:
I'll add a warning message here.